content/building-blocks/automate-and-execute-your-work/reproducible-work/google_cloud_docker.md

```diff
@@ -22,9 +22,16 @@ In order to be able to follow this building block you will need that the project
 
 {{% /warning %}}
 
-### Step 1: Install and Set up docker in a Google cloud instance
+### Step 1: Install and Set up Docker in a Google cloud instance
 
-Unless you have specified it otherwise, at the time of creating your instance Google cloud will use an image of the Debian Linux distribution as base OS for the instance. Accordingly you will have to follow the instructions on how to install the docker engine for this specific Linux distribution. If you want to check a more detailed step-by-step version of these you can visit [the offical Docker documentation](https://docs.docker.com/engine/install/debian/), where you will also find instructions for installing Docker on other Linux distributions and operative systems through the command line. If on the contrary you prefer to cut it straight to the chase, you can simply execute one by one the code lines below on your instance's command line to get Docker installed.
+The first thing you need to consider in order to install Docker in your instance is which boot disk image is it using. You can check a detailed version of the installation procedure in [the offical Docker documentation](https://docs.docker.com/engine/install/debian/), where you will find instructions for installing Docker on all supported Linux distributions and operative systems through the command line. If on the contrary you prefer to cut it straight to the chase, you can simply execute one by one the code lines below on your instance's command line to get Docker installed. These cover the installation of Docker for Ubuntu and Debian boot disk images. The instructions for these two Linux distributions are essentially the same except for steps #5 and #6 which have been duplicated to cover both, you have to pick the one that adjusts to your case.
+
+
+{{% tip %}}
+
+While Debian is the default boot disk image in Google cloud instances, we strongly recommend you to consider using a Ubuntu image if you are include a GPU to your instance. 

```

Comment:
I had a doubt here @narendramukherjee, should I use "strongly recommend" to reffer to the use of an Ubuntu image or instead use a different term? Is there a more precise way to address this? Your mention about using an Ubuntu image in last week's meeting was a recommendation in the sense that Ubuntu images are able to work better with the GPUs?, or instead and obligation in the sense that there is some actual barrier for container's use of GPU within intances using Debian images?

Reply:
I think we can say that "we recommend the usage of Ubuntu images because the installation of NVIDIA container toolbox is easy on Ubuntu".